spec/rails_helper.rb

```diff
@@ -0,0 +1,66 @@
+# rubocop:disable all
+# This file is copied to spec/ when you run 'rails generate rspec:install'
```

Comment:
While you did a good job working on the project requirements I noticed you disabled rubocop. It is against microverse policy to disable linters because linters help us write clean code. So it would be nice if you enabled rubocop